### DISCUSSION TITLE
When deploying page my font doesn't get recognized

### DISCUSSION THREAD

**Discussion by zvonimirkonjevic** [community/community#43424] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

When I deploy my project page it doesn't load font that is in project files. Help would be greatly appreciated. Thanks in advance.

Here is the link of page: https://zvonimirkonjevic.github.io/etch-a-sketch/

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @zvonimirkonjevic 

Your font is recognized by and computed by browser

![image](https://user-images.githubusercontent.com/47532703/210909650-26b869fc-39f4-44dc-a593-beee5c38b115.png)

**Reply by zvonimirkonjevic** [reply]
Hey thanks for responding, I noticed that it took a little longer to update the font but sadly couldn't delete my post. Anyway thanks again for your respond.
